src/auction/test/orders.test.mo

```diff
@@ -0,0 +1,291 @@
+import Prim "mo:prim";
+import Principal "mo:base/Principal";
+
+import Vec "mo:vector";
+
+import Auction "../src/lib";
+
+func init(trustedAssetId : Nat) : (Auction.Auction, Principal) {
```

Comment:
Each test duplicates same util functions. Perhaps it is worth putting them in a separate file to reuse

Reply:
Done